justfile

```diff
@@ -0,0 +1,74 @@
+set dotenv-load
+set positional-arguments
+
+# Adding a chain
+add-chain:
+	go run ./add-chain
+	go run ./add-chain check-rollup-config
+	go run ./add-chain compress-genesis
+	go run ./add-chain check-genesis
+	just codegen
+
+# Promote a standard candidate chain to a standard chain, example: just promote-to-standard 10
+promote-to-standard *args='':
+	go run ./add-chain promote-to-standard --chain-id=$1
+
+# Generate auto-generated files
+codegen:
+	go run superchain/internal/codegen/main.go
+	node ./scripts/codegen.js
+
+# Lint all Go code
+lint-all:
+	golangci-lint run superchain/... validation/... add-chain/... --fix
+
+# Test all Go code
+test-all: test-add-chain test-superchain test-validation
+
+# Test all Go code in the add-chain module
+test-add-chain:
+	# We separate the first test from the rest because it generates artifacts
+	# Which need to exist before the remaining tests run.
+	go test ./add-chain/... -run TestAddChain_Main -v
+	go test ./add-chain/... -run '[^TestAddChain_Main]' -v
+	just clean-add-chain
```

Comment:
I think its nice to leave these test files around for sanity checking so I'd prefer to remove this line and just manually run `just clean-add-chain` when needed. Those files are already .gitignored so it shouldn't cause issues

Reply:
[e44827e](https://github.com/ethereum-optimism/superchain-registry/pull/366/commits/e44827e28025e0e6a7a50413d5b27ad25f2026be)